### ISSUE TITLE
Issues with SAML

### ISSUE THREAD

**Issue by shrynx** [RocketChat/Rocket.Chat#4920]
The provided SAML configuration didn't work for me.
One issue was not supporting utf8 characters in XML. I have made a pull request for it. (https://github.com/RocketChat/Rocket.Chat/pull/4919)

The second issue is regarding RequestedAuthnContext@Comparison attribute. This is currently set to ```exact``` but our infrastructure required it to be ```better```.
There is no option to configure it in admin panel, but i think there should be one. For now we had to manually change and maintain the package.
```js
// packages/meteor-account-saml/saml_util.js 
// 100
'<samlp:RequestedAuthnContext xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol" Comparison="better">' 
```

**Comment by graywolf336** [comment]
@shrynx What are valid options for that value?

**Comment by shrynx** [comment]
@graywolf336 Yup, i referenced the same article. Perhaps using node package for saml2 https://github.com/Clever/saml2 would be more helpful. It exposes a lot more configuration options including the one i used here.
